### DISCUSSION TITLE
capa gsoc 2023 project description

### DISCUSSION THREAD

**Discussion by ggold7046** [mandiant/capa#1385] [General]
Hi , @williballenthin , @mr-tz 
I am having a tough time to write about project details of  `capa: Capabilities from Dynamic Analysis` .  Could you please guide me ?
And, there is this `weekly timeline` which seems a little confusing.

**Comment by mr-tz** [comment]
That's one of the core parts of your application and while we're happy to review your proposal we cannot write this for you.
The weekly timeline is a rough project plan on what you expect to work on each week.

**Comment by ooprathamm** [comment]
Respected FLARE team,

I have submitted a proposal for Project capa: Ghidra Integration on GSoC program website. 
I have added the link to the Google doc of my proposal to the top of my DRAFT pdf for remarks and comments.
It would be really helpful if the respected mentors (@mike-hunhoff ) could review it and provide their feedback and deep insights.

Thank you for your time.
Pratham

**Reply by mike-hunhoff** [reply]
@ooprathamm I enjoyed reading your proposal and left comments on the linked Google document. We look forward to receiving your submission via the GSoC website!

**Reply by ooprathamm** [reply]
@mike-hunhoff Thanks for providing the feedback so fast, will make sure to improve as per them before submitting.

**Comment by ggold7046** [comment]
> Hi , @williballenthin , @mr-tz I am having a tough time to write about project details of `capa: Capabilities from Dynamic Analysis` . Could you please guide me ? And, there is this `weekly timeline` which seems a little confusing.

Hi, I have sent a tentative application draft to flare-floss, could you kindly review and revert back.
